MetaMask/metamask-mobile
# feat(2796):  permission settings for multichain, behind feature flag

## **Description**
Hidden behind a feature flag, this is a new section in settings with the purpose of displaying all permissions, from in app dapp browser, wallet connect and SDK. This is a mocked UI version so that this part is done and available.





## **Related issues**

Contributes to solve issue: https://github.com/MetaMask/MetaMask-planning/issues/2796

## **Manual testing steps**

1. Go to settings by clicking the settings button in the bottom bar, then scroll to the Permissions item in the settings list.
2. After clicking the list item, the list of mocked permissions should be displayed.
5. These permissions present two dapps, uniswap and pancake swap, each connected via 3 different sources (Wallet Browser, SDK, and Wallet Connect). The connection sources are represented by tags in each list item.

## **Screenshots/Recordings**


| Before       | After        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/ee64634a-f76d-4012-b42f-10a30ee1fcf3"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/2fc7fef3-fa3a-4e79-b47d-0427ea308f11"> |









## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
